SPECS/python-process-tests/python-process-tests.spec

```diff
@@ -0,0 +1,111 @@
+%global srcname process-tests
+
+Summary:        Tools for testing processes
+Name:           python-%{srcname}
+Version:        2.0.2
+Release:        10%{?dist}
+License:        BSD
+URL:            https://github.com/ionelmc/python-process-tests
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Source0:        https://pypi.python.org/packages/source/p/%{srcname}/%{srcname}-%{version}.tar.gz
+BuildArch:      noarch
+
+%description
+Tools for testing processes.
+
+%package -n python%{python3_pkgversion}-%{srcname}
+Summary:        Tools for testing processes
+BuildRequires:  python%{python3_pkgversion}-devel
+BuildRequires:  python%{python3_pkgversion}-setuptools
+%{?python_provide:%python_provide python%{python3_pkgversion}-%{srcname}}
+
+%description -n python%{python3_pkgversion}-%{srcname}
+Tools for testing processes for Python 3.
+
+%prep
+%setup -q -n %{srcname}-%{version}
+
+%build
+%{py3_build}
+
+%install
+%{py3_install}
+
+%files -n python%{python3_pkgversion}-%{srcname}
+%doc README.rst
+%license LICENSE
+%{python3_sitelib}/__pycache__/*
+%{python3_sitelib}/process_tests*
+
+%changelog
+* Wed Jun 23 2021 Rachel Menge <rachelmenge@microsoft.com> - 2.0.2-10
+- Update cgmanifest and license info
```

Comment:
>Update cgmanifest [](http://example.com/codeflow?start=2&length=17)

Same comment about cgmanifest. Please update other specs, if it's going to turn out it's unnecessary.